docs_source_files/content/grid 4/_index.en.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Grid 4"
+chapter: true
+weight: 9
+---
+
+# Grid 4
+
+_Selenium Grid 4_ is a new smart proxy server
+that allows Selenium tests to route commands to remote web browser instances.
+Its aim is to provide an easy way to run tests in parallel on multiple machines.
+
+With Selenium Grid 4,
+one server acts as the hub that routes JSON formatted test commands
+to one or more registered Grid nodes.
+Tests contact the hub to obtain access to remote browser instances.
+The hub has a list of registered servers that it provides access to,
+and allows control of these instances.
+
+Selenium Grid 4 allows us to run tests in parallel on multiple machines,
+and to manage different browser versions and browser configurations centrally
+(instead of in each individual test).
+
+Selenium Grid 4 is not a silver bullet.
+It solves a subset of common delegation and distribution problems,
+but will for example not manage your infrastructure,
+and might not suit your specific needs.
```

Comment:
Hi @raju249 ,

Can you please changes in other tanslated pages other than en (like es,fr,de etc)

Thanks,
 Harsha